simplicity-sys/depend/wrapper.c

```diff
@@ -10,7 +10,7 @@
 
 #include <stdalign.h>
 
-const uint64_t c_overhead = overhead;
+const uint64_t c_overhead = 100;
```

Comment:
this was necessary to resolve `[C2099: initializer is not a constant` ... 

happy to change it if there's a better way, but all the stuff i tried resulted in the same issue 

https://stackoverflow.com/questions/6131455/compile-error-c2099-initializer-is-not-a-constant

Reply:
We will have to keep `c_overhead` in sync with `overhead`. Ideally this is automated with a script.

Reply:
Agreed, I've added [a change upstream](https://github.com/BlockstreamResearch/simplicity/pull/213/commits/d3baff3578e361032d8732e2ba41d8459cc9d34e#diff-07c9188115e9cbfbfa2a581496c14eee1c64e590c13bc6a8bc2fccd1b1bdebbbR44) that works for this. So once [#213](https://github.com/BlockstreamResearch/simplicity/pull/213) is merged and we can revendor simplicity here, this will work. 

```suggestion
const uint64_t c_overhead = overhead_value;
```

Reply:
fixed upstream, now unnecessary